.gitignore

```diff
@@ -63,3 +63,47 @@ test/unit/test_devices.py
 .vagrant
 
 tags
+### JetBrains template
```

Comment:
Can you also remove all of this? Most of it is completely irrelevant to OSS. It's fine if you want to exclude `.idea` but do is as a single line and we don't use jira, crashlytics or most of the stuff you added here so I'd appreciate if you could remove it. Thanks.